modules/ROOT/examples/live-demos/footnotes/index.js

```diff
@@ -0,0 +1,5 @@
+tinymce.init({
```

Comment:
These examples normally have the default CSS as well to make it easier for customers to copy. Take a look at checklists for example: https://github.com/tinymce/tinymce-docs/blob/6e25c6bfc9109a955891a56fd8e4b340cd169f1a/modules/ROOT/examples/live-demos/checklist/style.css

Reply:
Have added a `style.css` file as well. Thanks for the suggestion